demo/hparams/aevnmt_supervised_flickr.json

```diff
@@ -29,8 +29,8 @@
     "model_type": "aevnmt",
     "num_dec_layers": 1,
     "num_enc_layers": 1,
-    "num_epochs": 10,
-    "patience": 2,
+    "num_epochs": 15,
+    "patience": 5,
```

Comment:
Why did you change these? This is not what the downloaded demo models will be trained with, so it's better to leave them as they were.

Reply:
Accidental change.

Reply:
After you revert those feel free to merge.